lms/djangoapps/grades/api/views.py

```diff
@@ -185,7 +185,7 @@ def get(self, request, course_id):
             return grade_user
 
         prep_course_for_grading(course, request)
-        course_grade = CourseGradeFactory().create(grade_user, course)
+        course_grade = CourseGradeFactory().read(grade_user, course)
```

Comment:
Until the waffle flag is enabled, `read` will return `None` when the grade isn't found.  So, you'll need to update the code below to be:
```
course_grade.passed or None
course_grade.percent or None
course_grade.letter_grade or None
```